tools/theme-selector-plugin/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@sl-design-system/figma-theme-selector",
+  "license": "ISC",
+  "version": "1.0.0",
+  "description": "Figma Plugin to set the theme on a Figma file",
+  "main": "code.js",
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "build": "tsc -p tsconfig.json",
+    "lint": "eslint --ext .ts,.tsx --ignore-pattern node_modules .",
+    "lint:fix": "eslint --ext .ts,.tsx --ignore-pattern node_modules --fix .",
+    "watch": "npm run build -- --watch"
+  },
+  "devDependencies": {
+    "@figma/eslint-plugin-figma-plugins": "*",
+    "@figma/plugin-typings": "*",
+    "@typescript-eslint/eslint-plugin": "^6.12.0",
+    "@typescript-eslint/parser": "^6.12.0",
+    "eslint": "^8.54.0",
```

Comment:
We use version 9 in the rest of the repo, would be nice to use that here as well